src/frontend/src/pages/ProjectsPage/ProjectsTable.tsx

```diff
@@ -162,7 +167,21 @@ const ProjectsTable: React.FC = () => {
             quickFilterProps: { debounceMs: 500 }
           }
         }}
+        onFilterModelChange={(filterModel: GridFilterModel) => {
+          localStorage.setItem('projectsTableFilters', JSON.stringify(filterModel.items[0]));
+        }}
         initialState={{
+          filter: {
```

Comment:
All the code you wrote is really great. Especially the usage of the GridFilterModel!